src/Esfa.Roaao.Xslx.IntegrationTests/RoaaoExcelTests.cs

```diff
@@ -31,5 +33,47 @@ public void EpaShouldExist()
                 Assert.IsTrue(results.Organisations.Any(x => x.EpaOrganisationIdentifier == standardOrganisationsData.EpaOrganisationIdentifier), $"Couldn't find the Assessment Org for {standardOrganisationsData.EpaOrganisationIdentifier}");
             }
         }
+
+        [TestMethod]
```

Comment:
good spot, because it's packaged and run from VSTS, if we wanted to use NUnit you'd have to add the nunit adapter with the tests